tests/admin/test_u2f.py

```diff
@@ -0,0 +1,60 @@
+from .. import util
+import duo_client.admin
+from .base import TestAdmin
+
+
+class TestU2F(TestAdmin):
+    def test_get_user_u2ftokens(self):
```

Comment:
This should probably go in a different file, test_user_u2ftokens.py

Reply:
on it